cluster/common.sh

```diff
@@ -1080,21 +1090,32 @@ function generate-certs {
     ./easyrsa --dn-mode=org \
       --req-cn=kubecfg --req-org=system:masters \
       --req-c= --req-st= --req-city= --req-email= --req-ou= \
-      build-client-full kubecfg nopass) &>${cert_create_debug_output} || {
-    # If there was an error in the subshell, just die.
-    # TODO(roberthbailey): add better error handling here
+      build-client-full kubecfg nopass) &>${cert_create_debug_output} || true
+  if [ ! -s "${CERT_DIR}/pki/private/ca.key" \
```

Comment:
This doesn't have the same behavior. It's possible for the subprocess to fail in a way that is not caught be this iff. Should we loop over these files only to improve the error message?

Reply:
It definitely does not have the same behavior as what roberthbailey **intended** - that's why I wrote "**some** error handling". It's still better than what is there currently, i.e. no error detection at all.

I'm afraid adding bulletproof error handling here (in a way which still hides output from these commands unless there was an error) will be quite hard, given that we're calling bash functions and some of them are changing variables used elsewhere. That's why I did what I did, hoping that it will at least be a step towards solving #55229.